lib/_http_server.js

```diff
@@ -223,7 +224,8 @@ function writeHead(statusCode, reason, obj) {
   }
 
   if (common._checkInvalidHeaderChar(this.statusMessage))
-    throw new Error('Invalid character in statusMessage.');
+    throw new errors.Error('ERR_ASSERTION', 'Invalid character in ' +
+                    'statusMessage.');
```

Comment:
Likewise, `ERR_ASSERTION` is the wrong choice here. A new error code would be better.